Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_aitdm.nut

```diff
@@ -1,22 +1,36 @@
 untyped
 global function GamemodeAITdm_Init
 
-const SQUADS_PER_TEAM = 3
+// these are now default settings
+const int SQUADS_PER_TEAM = 4 // was 3
```

Comment:
Wait, if I'm reading this correctly, it previously only spawned 3 grunts per pod/dropship? o_O

Reply:
`SQUADS_PER_TEAM` determines how many AI squads there are per team, each squad has 4 npcs

Reply:
Correct